docs/benchmark.md

```diff
@@ -0,0 +1,108 @@
+# Benchmark baseline results
+
+This document contains baseline benchmark results for Observatorium API Gateway under synthetic load.
```

Comment:
What are trying to benchmark here? I assume API Gateway right? So why we need Thanos behind it? You essentially measure Thanos latency not API Gateway latency I guess. So mocked thing might be better for this then.

Reply:
As long as the metrics measure gw latencies I prefer to use some real backend instead of mocking

Reply:
But GW latency includes backend latency XD You are just benchmarking Thanos here...

Reply:
That is true, what I can do it mock the backend _responses_, so that gateway would behave as it'd behave in real-life with minimal latency. This would give more accurate results. I'm gonna work on it. Thanks @bwplotka